internal/os/disk/api.go

```diff
@@ -157,8 +157,8 @@ func (DiskAPI) InitializeDisk(diskNumber uint32) error {
 	return nil
 }
 
-func (DiskAPI) PartitionsExist(diskNumber uint32) (bool, error) {
-	cmd := fmt.Sprintf("Get-Partition | Where DiskNumber -eq %d", diskNumber)
+func (DiskAPI) BasicPartitionsExist(diskNumber uint32) (bool, error) {
+	cmd := fmt.Sprintf("Get-Partition | Where DiskNumber -eq %d | Where Type -eq Basic", diskNumber)
```

Comment:
@wongma7 if it's a MBR disk, `Type` would be `IFS`, so this change would break existing MBR disk